core-bundle/contao/templates/twig/content_element/accordion.html.twig

```diff
@@ -0,0 +1,39 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+
+{% block content %}
+    {% for element in elements %}
+        <h3 class="handorgel__header">
+            <button class="handorgel__header__button">{{ element.header }}</button>
+        </h3>
+        <div class="handorgel__content">
+            <div class="handorgel__content__inner">
+                {{ content_element(element.reference) }}
+            </div>
+        </div>
+    {% endfor %}
+{% endblock %}
+
+{% block script %}
+    {% add "handorgel_js" to body %}
+        <script src="{{ asset('js/handorgel.min.js', 'contao-components/handorgel') }}"></script>
+        <script>
+            (function () {
+                const accordion = new handorgel(document.querySelector('.content-accordion'), {
+                    {% block init_options %}
+                        // Put custom options here
+                        multiSelectable: false,
+                    {% endblock %}
+                });
+
+                // Show the first element
+                accordion.folds[0].open();
+            })();
+        </script>
+    {% endadd %}
+{% endblock %}
+
+{% block style %}
+    {% add "handorgel_css" to head %}
+        <link rel="stylesheet" href="{{ asset('css/handorgel.min.css', 'contao-components/handorgel') }}">
```

Comment:
Should we use the stylesheet component here like we do in the code content element? Then people have one place to adjust things and we could use lazy loading.

Reply:
I deliberately did not use it, because I think the concept is wrong. See https://stackoverflow.com/a/65984632 and scroll down to **I recommend you do NOT ever "preload" CSS for the following reasons!**. We should discuss this again in a separate issue.

Reply:
I'm not sure I agree with this post but we should definitely discuss how we want to handle these things. :+1: 
Using the component (even with lazy turned off) might still be a viable option, because then our users can decide if they want to use it or not by adjusting the component. :slightly_smiling_face:

Reply:
Changed in b4b454f60f04d827622bb8617fe6828785f493fc.